daemons/fenced/fenced_commands.c

```diff
@@ -1127,7 +1144,7 @@ status_search_cb(int pid, int rc, const char *output, void *user_data)
 
     mainloop_set_trigger(dev->work);
 
-    switch (rc) {
+    switch (result->exit_status) {
```

Comment:
should we take the exit_status directly here or rather encapsulate with stonith__result2rc?

Reply:
Fence agent status actions have their own exit code scheme, so no mapping before or after this PR -- it has its own enum

Reply:
aah crap ... sry I remember now.
was tempting to use something that doesn't rely on exit_status being initialized properly while in flight ;-)
just to be robust against issues where the termination might not be handled properly.